base_edifact/models/edifact.py

```diff
@@ -242,3 +242,47 @@ def map2odoo_description(self, seg):
             description = seg[2][3]
             return description
         return None
+
+    def _replace_empty(self, value):
```

Comment:
```suggestion
    def _safe_segment_element(self, value):
```

Reply:
Thank you @simahawk for your review. I updated it.